src/HowItWorks/HowItWorksScreen.tsx

```diff
@@ -226,11 +227,11 @@ interface PositionDotsProps {
   screenNumber: number
 }
 
-const NUMBER_OF_ONBOARDING_SCREENS = 5
-
 const PositionDots: FunctionComponent<PositionDotsProps> = ({
   screenNumber,
 }) => {
+  const howItWorksData = useHowItWorksData()
```

Comment:
its a bit odd to have this hook in the helper component, what do you think about parameterizing this?
```suggestion
  const PositionDots: FunctionComponent<PositionDotsProps> = ({
    highlightedDotIdx,
    totalDotCount
}) => {
```

Reply:
Sounds good!